pkg/helm/fake/fake_actions.go

```diff
@@ -17,6 +18,10 @@ func InstallChart(ns string, name string, url string, values map[string]interfac
 	}, nil
 }
 
+func InstallChartFailed(ns string, name string, url string, values map[string]interface{}, conf *action.Configuration) (*release.Release, error) {
```

Comment:
I would move these functions close to the tests, i.e. into the very same test file.